fe/fe-core/src/test/java/com/starrocks/sql/optimizer/rule/transformation/materialization/MvRewriteTest.java

```diff
@@ -1222,19 +1262,15 @@ public void testUnionRewrite() throws Exception {
             String plan8 = getFragmentPlan(query8);
             PlanTestBase.assertContains(plan8, "join_agg_union_mv_2");
             PlanTestBase.assertContains(plan8, "4:HASH JOIN\n" +
-                    "  |  join op: LEFT OUTER JOIN (BUCKET_SHUFFLE)\n" +
-                    "  |  colocate: false, reason: \n" +
-                    "  |  equal join conjunct: 20: v1 = 24: t1d\n" +
-                    "  |  \n" +
-                    "  |----3:EXCHANGE");
+                    "  |  join op: INNER JOIN (BUCKET_SHUFFLE)");
```

Comment:
why could convert left join to inner join?

Reply:
fixed.